### ISSUE TITLE
Checkboxes do not receive keyboard focus

### ISSUE THREAD

**Issue by Stegosource** [keystonejs/keystone#1908]
# Bug report

## Describe the bug

When I add a checkbox field to the admin UI, I cannot get to it via keyboard only navigation (ie. `tab` key). This is a minor inconvenience to people that prefer keyboard navigation in forms, but a major accessibility issue for anyone that does not have use of a mouse.

## To Reproduce

1. Create a new list with a checkbox
```
keystone.createList("User", {
  fields: {
    //...
    isAdmin: { type: Checkbox },
  }
})
```
2. Navigate to your admin dashboard (http://localhost:3000/admin/)
3. Click the add new User button
4. Using only the keyboard, try tabbing through all the inputs in the form.

## Expected behaviour

The "Is admin" checkbox should be focusable.

## System information

- OS: Ubuntu 19.10

## Additional context

I'm happy to help fix this issue. I tried looking into the admin UI code base, but did not find the file that I needed. However, I did see the culprit. The reason it's not reachable is because there is a toggle of the `tabindex` on the checkbox. For some reason, it has a value of `-1` if it's unchecked, and `0` when checked. This may serve a purpose that I am not thinking of, but for the sake of this conversation, just removing the tabindex key entirely should fix the issue.

**Comment by MadeByMike** [comment]
@Stegosource No arguments here. This is important to us. I'll try and get some time to fix it in the next few days.

**Comment by gautamsi** [comment]
@MadeByMike is #1913 related?

**Comment by Stegosource** [comment]
@gautamsi I looked at the changes and it definitely addresses the tabindex issue as well as provides even better a11y features.
